.travis.yml

```diff
@@ -22,7 +22,11 @@ matrix:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
+  - mkdir travis-phantomjs
```

Comment:
in v4.0.0 I've got plans to dump phantom for firefox using something like this

https://github.com/martndemus/ember-cli-font-awesome/commit/0ba4900d1fd7aa1d37aa1b8bf2b1336a089625e8

I honestly haven't used phantomjs in the last year and it's not fun to debug/no customers are using it in production/ etc